src/views/vehicle/view_vehicle.cpp

```diff
@@ -89,6 +91,89 @@ namespace big
 			components::command_checkbox<"vehiclecontrol">();
 			components::command_checkbox<"keepvehicleclean">();
 
+			ImGui::Checkbox("Custom Ammo", &g.vehicle.vehicle_ammo_special.toggle);
```

Comment:
This should be under vehicle -> Fun Options, as this does not apply to or relate to just the vehicle itself and is situational at best, because some vehicles do not have it.

Reply:
I was struggling between self->weapons, vehicle->general, or vehicle->weapons(new tab). I agree now that fun options is a good place for it.